### PR TITLE
SVCPLAN-2993 Turn on verbose SFTP sshd subsystem by default

### PR DESCRIPTION
**Pull request by bsper2**
NOTE: If you manually turned on verbose SFTP sshd subsystem in your project's control repo by setting:
sshd::config_subsystems:
  sftp: "/usr/libexec/openssh/sftp-server -l VERBOSE"
You can now remove that

Also update pdk version to 2.7.1 to get github action working again

### CODE REVIEW

**Comment by bsper2** [comment]
Tested on Radiant

The changes to the `data/os/RedHat.yaml` file is the only change related to the SFTP change, all others are related to updating PDK

File: data/os/RedHat.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ sshd::config:
   PubkeyAuthentication: "no"
   UsePAM: "yes"
 sshd::config_subsystems:
-  sftp: "/usr/libexec/openssh/sftp-server"
+  sftp: "/usr/libexec/openssh/sftp-server -l VERBOSE"
```
Comment (billglick) [inline data/os/RedHat.yaml:12]: This is the only real functional change, correct?

All other changes are related to linting or the pdk update.

**Review by billglick — APPROVED** [review]
This looks good.

File: data/os/RedHat.yaml (hunk 1 above)
Comment (bsper2) [reply data/os/RedHat.yaml:12]: Correct yeah, that line is the only real change here.

**Pull request merged**